r/R/arrow-package.R

```diff
@@ -27,7 +27,7 @@
 #' @importFrom rlang is_list call2 is_empty as_function as_label arg_match is_symbol is_call call_args
 #' @importFrom rlang quo_set_env quo_get_env is_formula quo_is_call f_rhs parse_expr f_env new_quosure
 #' @importFrom rlang new_quosures expr_text
-#' @importFrom tidyselect vars_pull vars_rename vars_select eval_select
+#' @importFrom tidyselect vars_pull vars_select eval_select eval_rename
```

Comment:
Do we need to update the other usages of vars_select in the package too?

Reply:
Yep - this is still on my to-do list, but I think all other feedback has been addressed.

Reply:
Done now